UM-USElab/gradecraft-development
# Froala Update

### Status
**READY**

### Description
This PR removes the sanitize configuration of Froala and fixes it so that the HTML and video buttons (previously added, but not working) show.

### Deploy Notes
Confirm that froala works as expected on both the `Grade` form and any other usage of the froala form (Assignment, Badge, etc)

### Migrations
NO

### Steps to Test or Reproduce
Load the Grade form, confirm that you see the video button and can edit the HTML code. 
Do the same for th


### Impacted Areas in Application
List general components of the application that this PR will affect:

* All froala forms
